com.unity.render-pipelines.core/Editor/MaterialUpgrader.cs

```diff
@@ -57,6 +65,36 @@ class KeywordFloatRename
         }
         List<KeywordFloatRename> m_KeywordFloatRename = new List<KeywordFloatRename>();
 
+        /// <summary>
+        /// Type of property to rename.
+        /// </summary>
+        public enum RenameType
+        {
+            /// <summary>Texture reference property.</summary>
+            Texture,
+            /// <summary>Float property.</summary>
+            Float,
+            /// <summary>Color property.</summary>
+            Color
+        }
+
+        /// <summary>
+        /// Retrieves a collection of renamed parameters of a specific RenameType.
+        /// </summary>
+        /// <param name="type">Rename Type</param>
+        /// <returns>Dictionary of property names to their renamed values.</returns>
+        /// <exception cref="ArgumentException">type is not valid.</exception>
+        public IReadOnlyDictionary<string, string> GetRename(RenameType type)
```

Comment:
```suggestion
        public IReadOnlyDictionary<string, string> GetPropertyNameMap(RenameType type)
```

Reply:
I'm not sure I really like that suggested name though. I think `GetRename` is closer to the truth in that instance.
`GetPropertyRenameMap` maybe? Ultimately, I'm not the major stakeholder here, so I will defer to your better judgment :)

Reply:
I like `GetPropertyRenameMap` better, seems more descriptive.

Reply:
Fixed in 3250edaf413e50e2d32a9a32a3f5eae82e4f9c29.